packages/core/src/profile/redux/actions/__tests__/doPatchUserAttribute.test.js

```diff
@@ -0,0 +1,81 @@
+import { mockPatchUserAttributeResponse } from '../../__fixtures__/userAttribute.fixtures';
+import { mockStore } from '../../../../../tests';
+import doPatchUserAttribute from '../doPatchUserAttribute';
+import find from 'lodash/find';
+import reducer, { actionTypes } from '../..';
+
+const profileMockStore = (state = {}) =>
+  mockStore({ profile: reducer() }, state);
+
+let store;
+
+describe('doPatchUserAttribute action creator', () => {
+  const userId = 123456;
+  const attributeId = '123456';
+  const data = {};
```

Comment:
it would be helpful in the future (either to debug or migrate to typescript) if you can use the real properties here instead of an empty object.